openwisp_monitoring/monitoring/migrations/0010_add_group_permissions.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('device_monitoring', '0004_wificlient_wifisession'),
+        ('monitoring', '0009_alter_alertsettings_options'),
```

Comment:
why this renaming here?

Reply:
Because I have moved `add_group_permissions` migration to `monitoring app`.